.github/actions/tests/run-api-tests/action.yml

```diff
@@ -6,6 +6,8 @@ inputs:
     report-name:
         description: Name of Allure report to be generated.
         required: true
+    tests:
+        description: Specific tests to run, separated by single whitespace. See https://playwright.dev/docs/test-cli
```

Comment:
There are cases where we only need to run one E2E spec file, like `update-woocommerce.spec.js` and `upload-plugin.spec.js`. This `tests` input would allow us to specify those spec file/s.